.github/workflows/bananapi-m1-plus.yml

```diff
@@ -0,0 +1,34 @@
+name: BananaPi-M1+
+
+on:
+  pull_request:
+    branches:
+      - main
+      - master
+      # ESR branches glob pattern
+      #- v20[0-9][0-9].[0-1]?[1470].[0-9]+
```

Comment:
Why are ESR branches enabled for PRT but not PR? These should always match.

I think we want ESR branches enabled for all devices being sanity tested here, not just devices that are actually released for ESR?